components/basejob/Basejobfragment.vue

```diff
@@ -3,11 +3,11 @@
     <pre>
     #!/bin/bash
     <fragment v-if="sbatch[sbatchIndex].jobname">#SBATCH --job-name {{ sbatch[sbatchIndex].jobname }} </fragment>
-    <fragment v-if="sbatch[sbatchIndex].partition">#SBATCH --partition {{ sbatch[sbatchIndex].partition }}</fragment>
-    <fragment v-if="sbatch[sbatchIndex].nnodes">#SBATCH --nodes {{ sbatch[sbatchIndex].nnodes }} </fragment>
-    <fragment v-if="sbatch[sbatchIndex].ncpus">#SBATCH --cpus-per-task {{ sbatch[sbatchIndex].ncpus }} </fragment>
-    <fragment v-if="sbatch[sbatchIndex].memory">#SBATCH --mem {{ sbatch[sbatchIndex].memory }} </fragment>
-    <fragment v-if="sbatch[sbatchIndex].time">#SBATCH --time {{ timeString }} </fragment>
+    <fragment >#SBATCH --partition {{ sbatch[sbatchIndex].partition }}</fragment>
+    <fragment >#SBATCH --nodes {{ sbatch[sbatchIndex].nnodes }} </fragment>
+    <fragment >#SBATCH --cpus-per-task {{ sbatch[sbatchIndex].ncpus }} </fragment>
+    <fragment >#SBATCH --mem {{ sbatch[sbatchIndex].memory }} </fragment>
+    <fragment >#SBATCH --time {{ timeString }} </fragment>
     <fragment v-if="sbatch[sbatchIndex].output">#SBATCH --output {{ sbatch[sbatchIndex].output }} </fragment>
     <fragment v-if="sbatch[sbatchIndex].emailevents.length > 0">
     #SBATCH --mail-type {{ emailEventString }} 
```

Comment:
For these v-ifs that were removed, do you want this to show all the time? Just checking!

Reply:
Could have a wider parent div like `<div v-if="sbatch[sbatchIndex]></div>`

Reply:
Great question! I realized that these inputs always have a value. So if I want to make them optional I need to think of a way to have a different validation or UI component to disable. So yes, this is intentional for now!